client/index.html

```diff
@@ -37,7 +37,7 @@
             var tr = document.createElement('tr');
             row.forEach((cell, index) => {
                 if(index==0){ // delete first column "id"
-                    row.deleteCell(0);
```

Comment:
I think you will need to modify line 30 of this file as we did for this function (i.e., the `forEach` call) to make sure that no `id` column is generated